Bio/PDB/StructureBuilder.py

```diff
@@ -185,15 +186,15 @@ def init_residue(self, resname: str, field: str, resseq: int, icode: str):
     def init_atom(
         self,
         name: str,
-        coord: np.array,
+        coord: np.ndarray,
```

Comment:
Why?

Reply:
[np.array](https://numpy.org/doc/stable/reference/generated/numpy.array.html) is not a type but a function which returns an [np.ndarray](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html).

mypy reports:
```
error: Function "numpy.core.multiarray.array" is not valid as a type  [valid-type]
```

Reply:
Thanks - I guess I'll get used to it, but I haven't been type annotating with numpy yet.

I presume that is long standing and doesn't need a particularly recent numpy dependency?

Reply:
> I presume that is long standing and doesn't need a particularly recent numpy dependency?

that's correct.